resources/todomvc/architecture-examples/angular/src/app/todo-header/todo-header.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { TodosService } from '../todos.service';
+
+@Component({
+  selector: 'app-todo-header',
+  templateUrl: './todo-header.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
Can you please explain in a comment why you need to change the `changeDetection` parameter here?

Reply:
removed from header, after running tests with no measurable difference